hedera-node/src/main/java/com/hedera/services/state/expiry/renewal/RenewalProcess.java

```diff
@@ -101,8 +101,9 @@ private EntityProcessResult renewIfTargeted(final EntityNum entityNum, final boo
 		final long renewalPeriod = assessment.renewalPeriod();
 		final long renewalFee = assessment.fee();
 		helper.renewLastClassifiedWith(renewalFee, renewalPeriod);
-		recordsHelper.streamCryptoRenewal(
-				entityNum, renewalFee, lastClassified.getExpiry() + renewalPeriod, isContract);
+		final var payer = helper.getPayerForAutoRenew();
```

Comment:
Can we use this `payer` account as the first argument to `fees.assessCryptoAutoRenewal()` [here](https://github.com/hashgraph/hedera-services/blob/eth-tx-interop/hedera-node/src/main/java/com/hedera/services/state/expiry/renewal/RenewalProcess.java#L100)?

I ask because that method takes the payer's balance into account. If the payer can only afford a "partial renewal", then the returned `assessment` is for the partial renewal.

Reply:
Got it ! Made the changes

Reply:
It looks like we will use `accountRenewalPrices` if we use `payer` as first argument in `renewalFees` method [here](https://github.com/hashgraph/hedera-services/blob/3162-M-support-auto-renew-accounts/hedera-node/src/main/java/com/hedera/services/fees/calculation/AutoRenewCalcs.java#L141). I think we should add additional argument `payer` use the `payer's balance` only in `assess` method. What do you think ?

Reply:
Ah yes, that makes perfect sense to me.